sktime/datasets/tests/test_datadownload.py

```diff
@@ -91,3 +93,12 @@ def test_load_forecasting_data_invalid_name(name):
         match=f"Error in load_forecastingdata, Invalid dataset name = {name}.",
     ):
         load_forecastingdata(name=name)
+
+@pytest.mark.datadownload
+def test_load_fpp3():
```

Comment:
could you cycle over a random subsample, like `test_check_link_downloadable`?

Reply:
I read the code in test_check_link_downloadable. I don't see what that has to do with a random subsample, and I don't see why this would be helpful. The test I provided exercises the full functionality of load_fpp3(), on a specific example, which I chose carefully to be sufficiently complex (pd_multiiindex_hier) but also a small dataset to minimize network traffic and processing time, to speed up the test.

Reply:
ok, I suppose that's good enough for the initial PR.

I will not consider this a blocking issue (unlike the code formatting failure 😁 ), the general rationale to check all datasets once in a while is that upstream providers sometimes rename them or change links, for individual datasets only.

However, given that we are interfacing another package, the risk is probably relatively low, so I do not mind not doing this, as the upstream package surely is testing that extensively.